packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -640,11 +644,19 @@ const Combobox = forwardRef<
     );
 
     const errorId = `${id}-error`;
+    const descriptionId = `${id}-description`;
     const inputProps = {
       ...props,
-      'aria-describedby': error
-        ? addIdRef(ariaDescribedby, errorId)
-        : ariaDescribedby
+      'aria-describedby': (() => {
```

Comment:
Is there a reason this needs to be a self-invoking function and we can't just have the value set via a variable above?

Reply:
Nope. Goog call out. I did do it as ifs so it is not a double ternary.